File: test_hermit.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch(headless=False)
    page = browser.new_page()
    
    print('1. 访问页面...')
    page.goto('http://localhost:18080')
    page.wait_for_timeout(2000)
    
    print('2. 检查下拉框...')
    options = page.locator('#agentType option').all_text_contents()
    print('选项:', options)
    
    print('3. 输入名称并点击创建...')
    page.fill('#agentName', 'pw-final-test')
    page.click('#createBtn')
    page.wait_for_timeout(5000)
    
    notice = page.locator('#notice').text_content()
    print('结果:', notice)
    
    page.screenshot(path='/tmp/hermit-final.png', full_page=True)
    print('截图保存到: /tmp/hermit-final.png')
    
    browser.close()
    print('测试完成!')

File: test-hermit.mjs
import { chromium } from 'playwright';

const browser = await chromium.launch({ headless: false });
const page = await browser.newPage();

console.log('1. 访问页面...');
await page.goto('http://localhost:18080');
await page.waitForTimeout(2000);

console.log('2. 检查下拉框...');
const options = await page.locator('#agentType option').allTextContents();
console.log('选项:', options);

console.log('3. 输入名称并点击创建...');
await page.fill('#agentName', 'playwright-test');
await page.click('#createBtn');
await page.waitForTimeout(3000);

const notice = await page.locator('#notice').textContent();
console.log('结果:', notice);

await page.screenshot({ path: '/tmp/hermit-result.png', fullPage: true });
console.log('截图: /tmp/hermit-result.png');

await browser.close();
console.log('完成!');
